src/Couchbase.Lite.Support.UWP/Couchbase.Lite.Support.UWP.csproj

```diff
@@ -241,7 +241,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Couchbase.Lite\Couchbase.Lite.csproj">
-      <Project>{dd9cb298-f3a1-4653-883b-352e69c16f93}</Project>
+      <Project>{a2fee0b2-73c3-4687-8fbc-3c8e0cde3432}</Project>

```

Comment:
It would be better to get revert it back to the previous GUID to remove this entirely from the diff